### PR TITLE
feat: epd-visualization integration library

### PR DESCRIPTION
**Pull request by sam-garland**
Changed branch name as requested: `epic/epd-visualization-maintenance` -> `integration-libs/epd-visualization-maintenance`

Previous PR before branch change: https://github.com/SAP/spartacus/pull/14420

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,32 +14,36 @@ branches:
...
-  - script:
-    name: "E2E tests for B2C, 4th job"
-    <<: *defaults
-  - script: "./ci-scripts/e2e-cypress.sh --ssr"
-    name: "E2E tests for SSR B2C"
-  - script: "export CYPRESS_CONFIG=cypress.ci.cdc.json && export SPA_ENV='cdc' && ./ci-scripts/e2e-cypress.sh -s cdc || exit 0"
-    name: "E2E tests for CDC"
-  - script: "./ci-scripts/lhci.sh"
-    name: "Lighthouse score validation"
+  include:
```
Comment (hackergil) [inline .travis.yml:17]: Why was `jobs` replaced with `include` ?

**Review by hackergil — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (sam-garland) [reply .travis.yml:17]: `jobs:` is still present on line 16.

I am using the syntax documented for conditional jobs here:
https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditional-jobs

The page linked above contains the following text:
`Jobs need to be listed explicitly, i.e., using jobs.include (or its alias matrix.include), in order to specify conditions for them.`

Comment (hackergil) [reply .travis.yml:17]: Got it. Thanks!

**Review by Platonn — APPROVED** [review]

**Comment by sam-garland** [comment]
Thanks. I will advise when the documentation has been merged/published.

**Pull request merged**